webapp/channels/src/components/admin_console/admin_settings.tsx

```diff
@@ -15,7 +15,7 @@ import FormError from 'components/form_error';
 import AdminHeader from 'components/widgets/admin_console/admin_header';
 
 export type BaseProps = {
-    config?: DeepPartial<AdminConfig>;
+    config: DeepPartial<AdminConfig>;
```

Comment:
lets keep this as it and define config in Props
follow https://github.com/mattermost/mattermost/blob/8056be1d165c68e19dcda7429001055af9489d75/webapp/channels/src/components/admin_console/bleve_settings.tsx#L23-L25